### ISSUE TITLE
Some 78.x flags (e.g. Wellbeing, Google Lens, Password leaks check)

### ISSUE THREAD

**Issue by kowith337** [bromite/bromite#396]
**Is your feature request related to privacy?**
Probably **Yes**, maybe.

**Is there a patch available for this feature somewhere?**
Don't know

**Describe the solution you would like**
Around Chromium 78, I've seen this flag appear
- `#usage-stats` Share Usage Stats with Digital Wellbeing
- `#password-leak-detection` Password Leak Detection
- `#context-menu-search-with-google-lens` Google Lens powered image search in the context menu

With my self concern...
- Are the Digital Well(`sh**`)being will be turn on by default in upstream app right now or in the future? (e.g. All of Chrome variants, Beta, Dev, Chromium Vanilla, Brave, etc. except this repo and [Marmaduke](https://github.com/macchrome/droidchrome)'s fork), also if it turn on, it will attempt to phoning home?
- Are password leaks feature contact to home server or somewhere?
- Are "Google Lens" feature will binding together with Chrome and GMS? (aka. Google Play Services)
- In other words, are those features attempt to borrow, use, open or contacting with external apps above?

**Describe alternatives you have considered**

**Comment by csagan5** [comment]
> * also if it turn on, it will attempt to phoning home?

In cases like this you should test and report back, you should not trust anybody; even if I told you, I might be wrong because not everything is 100% tested.

In this specific case I am pretty sure that it can't work for layered reasons:
* there is no API key
* we have "kill" patches to disable Play Services integrations, including GMS
* I remember checking that this functionality was off while adapting the patches to v78

> * Are password leaks feature contact to home server or somewhere?

It would contact `https://passwordsleakcheck-pa.googleapis.com/v1/leaks:lookupSingle` but as explained in https://chromium.googlesource.com/chromium/src/+/HEAD/components/password_manager/core/browser/leak_detection/leak_detection_request.h it would require an access token, which is not available because there is no API token.

This feature is disabled by default in Chromium; the password manager is disabled by default in Bromite.

> * Are "Google Lens" feature will binding together with Chrome and GMS? (aka. Google Play Services)

No, I think it's equivalent to a contextual menu search, but I have not investigated; it is also disabled by default in Chromium.

> * `#usage-stats` Share Usage Stats with Digital Wellbeing

This one is enabled by default, it works on Q+ but does not send data anywhere, it uses a local database from what I have seen. I will disable it by default on next release, just because I do not think it should be enabled by default.

> * In other words, are those features attempt to borrow, use, open or contacting with external apps above?

Play Services/Store integrations are disabled, including GMS. I exclude it, but please prove me wrong, it would be a good chance to improve the situation.

Also bear in mind that since there is no API key and no account association possibility we can be sure that even if a request reached some server, it would not be as useful as those which carry an user ID tag, I would expect such servers to not be able to proficiently use the information except for some metric somewhere reporting "weird"/unexpected requests.

**Comment by csagan5** [comment]
`#usage-stats` is disabled in current Bromite but it will be enabled in some future version as you can see here: https://chromium.googlesource.com/chromium/src/+/ad38303cf820cda1f6b5e31d27bd2bd58fc66571

When that happens I will put it back to disabled; however they cannot send this data online without asking for user consent because of GDPR, so even if enabled it must stay only on the local storage of Chromium/Bromite and cannot be shared with another app.

I think the only purpose is to have Android Q gather it to show wellbeing/usage stats; whether Android will send it somewhere, that I don't know and it's worth investigating.

**Comment by NomarCub** [comment]
I can't find #usage-stats in flags now. Can Digital Wellbeing be enabled for Bromite, and if so, how?
